CHANGELOG.md

```diff
@@ -1,13 +1,17 @@
 ## Changelog
 
-### 4.0.0-pp (2021-05-07)
+### 4.0.0 (2021-06-22)
+**Breaking change:** Config API contract has changed and now requires the v2 GET endpoint. The SDK will not work with the v1 endpoint.
 - Improvements:
+    - Updated company name references. [SDKCF-3550]
+	- Added handling for getConfig response status codes 400 and 404. [SDKCF-3884]
```

Comment:
This looks fine locally and in github's markdown view 🤷‍♀️

Reply:
Hmm, ok 👍